infra/cifuzz/cifuzz.py

```diff
@@ -201,6 +211,56 @@ def run_fuzzers(fuzz_seconds, workspace):
   return True, False
 
 
+def get_lastest_build_version(project_name):
+  """Gets the latest OSS-Fuzz build version for a projects fuzzers.
+
+  Args:
+    project_name: The name of the project who's build is being retrieve.
+
+  Returns:
+    A string with the latest build version or None.
+  """
+  http_get_string = GC_STORAGE + '{0}/{0}-address-latest.version'.format(
+      project_name)
+  try:
+    response = urllib.request.urlopen(http_get_string)
+  except urllib.error.HTTPError:
+    logging.error('Error getting the lastest build version for %s.',
+                  project_name)
+    return None
+  return response.read().decode('UTF-8')
+
+
+def download_old_build_dir(project_name, out_dir):
+  """Download an old OSS-Fuzz build to get an old version of fuzzers.
+
+  Args:
+    project_name: The name of the relevant OSS-Fuzz project.
+    out_dir: The location where the build should be stored.
+
+  Returns:
+    A path to where the old build is located, or None.
+  """
+  latest_build_str = get_lastest_build_version(project_name)
+  if not latest_build_str:
+    return None
+  if not os.path.exists(out_dir):
```

Comment:
check this first, as it doesn't require sending any http requests

Reply:
Fixed.